project/C3WebBuild.scala

```diff
@@ -27,75 +26,91 @@ object C3WebBuild extends Build {
-      }
-    )
+      buildSettings ++
+      Seq(
+        shellPrompt := {
+          s => Project.extract(s).currentProject.id + " > "
+        }
+      )
   }
 }
 
 object ProjectSettings {
-  val ProjectVersion = "1.0"
-  val ScalaVersion = "2.11.5"
-
   lazy val buildSettings = Seq(
     organization := "com.ifunsoftware.c3web",
     version := ProjectVersion,
     scalaVersion := ScalaVersion
   )
-
   lazy val defaultSettings = Defaults.defaultSettings ++
-      ScalastylePlugin.Settings ++
-      formatSettings ++
-  Seq(
+    ScalastylePlugin.Settings ++
+    formatSettings ++
+    Seq(
       scalacOptions in Compile := Seq(
         "-encoding", "utf8", "-target:jvm-1.8", "-feature", "-language:implicitConversions", "-language:postfixOps", "-unchecked", "-deprecation",
-        "-Ywarn-adapted-args", "-Xlog-reflective-calls"        
+        "-Ywarn-adapted-args", "-Xlog-reflective-calls"
       ))
-
+  resolvers ++=
+    Seq(
+      "spray repo" at "http://repo.spray.io/"
+    )
   lazy val c3webAssemblySettings = Seq(
     mainClass in assembly := Some("com.ifunsoftware.c3web.Boot"),
-    jarName   in assembly := "c3web-server.jar")
-
+    jarName in assembly := "c3web-server.jar")
   lazy val formatSettings = SbtScalariform.scalariformSettings ++ Seq(
     ScalariformKeys.preferences in Compile := formattingPreferences,
-    ScalariformKeys.preferences in Test    := formattingPreferences
+    ScalariformKeys.preferences in Test := formattingPreferences
   )
-
   lazy val formattingPreferences = {
     import scalariform.formatter.preferences._
     FormattingPreferences()
       .setPreference(AlignParameters, true)
       .setPreference(AlignSingleLineCaseStatements, true)
   }
+  val ProjectVersion = "1.0"
+  val ScalaVersion = "2.11.5"
 }
 
 object Dependencies {
-  val akkaV       = "2.3.12"
+  val akkaV = "2.3.12"
   val akkaStreamV = "1.0"
-  val scalaTestV  = "2.2.4"
-
-  object Compile {
-    val akkaActor       = "com.typesafe.akka" %% "akka-actor"                           % akkaV
-    val akkaStream      = "com.typesafe.akka" %% "akka-stream-experimental"             % akkaStreamV
-    val akkaHttpCore    = "com.typesafe.akka" %% "akka-http-core-experimental"          % akkaStreamV
-    val akkaHttp        = "com.typesafe.akka" %% "akka-http-scala-experimental"         % akkaStreamV
-    val sprayJson       = "com.typesafe.akka" %% "akka-http-spray-json-experimental"    % akkaStreamV
-    val akkaHttpTestkit = "com.typesafe.akka" %% "akka-http-testkit-scala-experimental" % akkaStreamV
-  }
-
-  object Test {
-    val scalatest = "org.scalatest"     %% "scalatest"      % scalaTestV   % "test"    
-  }
-
+  val scalaTestV = "2.2.4"
+  val sprayV = "1.3.3"
   val c3webServer = Seq(
     Compile.akkaActor,
     Compile.akkaStream,
     Compile.akkaHttpCore,
     Compile.akkaHttp,
     Compile.sprayJson,
+    Compile.sprayJSONNE,
     Compile.akkaHttpTestkit,
+    Compile.sprayServlet,
```

Comment:
why do we need sprayServlet dependency?) As far as I remember you need it only to run in a servlet container like Tomcat, Jetty and etc.